specification/netapp/resource-manager/Microsoft.NetApp/stable/2023-07-01/examples/Accounts_Get.json

```
@@ -0,0 +1,36 @@
+{
+  "parameters": {
+    "subscriptionId": "D633CC2E-722B-4AE1-B636-BBD9E4C60ED9",
+    "resourceGroupName": "myRG",
+    "accountName": "account1",
+    "api-version": "2023-07-01"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/D633CC2E-722B-4AE1-B636-BBD9E4C60ED9/resourceGroups/myRG/providers/Microsoft.NetApp/netAppAccounts/account1",
+        "name": "account1",
+        "type": "Microsoft.NetApp/netAppAccounts",
+        "location": "eastus",
+        "properties": {
+          "provisioningState": "Succeeded",
+          "activeDirectories": [
+            {
+              "site": "SiteName",
+              "activeDirectoryId": "02da3711-6c58-2d64-098a-e3af7afaf936",
+              "username": "ad_user_name",
+              "domain": "10.10.10.3",
+              "dns": "10.10.10.3, 10.10.10.4",
```

```suggestion
              "dns": "10.10.10.3",
```
please update to this